src/components/common/Map.js

```diff
@@ -1,31 +1,36 @@
 import React from 'react';
 import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
-import { bool } from 'prop-types';
+import { object, bool, func } from 'prop-types';
 
 import marker from '../../img/marker.svg';
 
-const Map = withScriptjs(withGoogleMap((props) => {
-  let latitude;
-  let longitude;
-  navigator.geolocation.getCurrentPosition((position) => {
-    ({ latitude, longitude } = position.coords);
-  });
-  const { isMarkerShown } = props;
+const Map = ({ handleClick, isMarkerShown, coords, target }) => {
+  const { latitude, longitude } = coords;
```

Comment:
You can move this destructuring to the signature of the method, and make this component even shorter with no curly brackets.

Reply:
How so?